translations/messages.en.yml

```diff
@@ -503,6 +503,10 @@ import:
         page_title: 'Import > elCurator'
         description: 'This importer will import all your elCurator articles.'
         how_to: Please select your elCurator export and click on the button below to upload and import it.
+    omnivore:
+        page_title: 'Import > Omnivore'
+        description: 'This importer will import all your Omnivore articles.'
+        how_to: Please select your Omnivore export and click on the button below to upload and import it.
```

Comment:
We should update the phrasing here because the importer does not support the zip file. We need to specify that we are expecting the metadata.json file